en/docs/services/create-your-first-service.md

```diff
@@ -1,177 +1,219 @@
 
-- In the code view
+To build the JSON payload and respond, follow these steps:
 
-  ![Code View](../assets/img/services/choreo-service-code-view.png){.cInlineImage-full}
+1. To build the `json` payload with data of the total cases per million in the population, add a variable.
 
+    Click the last **+** icon in the low-code diagram and click **Variable**. Then enter information as follows:
     
-To validate the service, click **Run & Test**. The following is logged to indicate that you have successfully started the service.
+    | **Field**      | **Value**                                         |
+    |----------------|---------------------------------------------------|
+    | **Type**       | `json`                                            |
+    | **Name**       | `payload`                                         |
+    | **Expression** | `{ TotalCasesPerMillion : totalCasesPerMillion }` |
+    
+    Save the information.
+    
+2. To respond with the JSON payload, add a `Respond` statement.
 
-![Service Started Log](../assets/img/services/service-started-notification.png){.cInlineImage-half}
+    Click the last **+** icon in the low-code diagram and click **Respond**.
     
-Now you are ready to test the service.
-   
-## Step 3: Test the service
+    In the **Respond Expression** field, enter `payload`.
+    
+    Save the information.
+    
+Now you have completed designing the `CovidStatus` service.
+
+## Step 6: Test the service
 
-To test the `covid-stats` service, follow this procedure:
+To test the `CovidStatus` service you created, follow the procedure below:
 
-1. Click the **Test** icon in the left pane.
+1. Click **Run & Test**.
+
+    The following logs appear to indicate that the service has successfully started.
+    
+    ```
+    Starting application...
+    [ballerina/http] started HTTP/WS listener 0.0.0.0:8090
+    ```
+   
+2. Click the **Test** icon in the left pane.
 
     ![Test Icon](../assets/img/services/test-icon.png){.cInlineImage-bordered}
 
-2. In the section that opens to the right of the page, click **GET**.
+3. In the test view that opens to the right of the page, click **GET**.
 
-3. Click **Try it out**, and then click **Execute**.
+4. Click **Try it out**, and then enter the following information in the fields that appear.
+
+    | **Field**     | **Value**                                         |
+    |---------------|---------------------------------------------------|
+    | **country**   | `USA`                                            |
+    | **date**      | `2019`                                         |
+   
+5. Click **Execute**.
 
-The following figure shows the server response to the search results for the United States COVID-19 statistics:
+    The response is displayed as follows:
 
-![Server Response](../assets/img/services/server-response.png){.cInlineImage-half}
+    ![Response in the test view](../assets/img/services/test-view-response.png){.cInlineImage-bordered}
```

Comment:
This image is not rendering in view  file mode. Should be some spacing issue at the beginnning.

Reply:
This image renders in the local build.

It seems like only merged images render in this view. There are a few other updated images included in this PR, but the images that show when you view it via the **View File** option are the older versions.

Reply:
test-view-response.png seems  no such image in the repo

Reply:
Added the image.